packages/editor-ui/src/components/WorkflowActivator.vue

```diff
@@ -1,19 +1,22 @@
 <template>
 	<div class="workflow-activator">
-		<el-switch
-			v-loading="loading"
-			element-loading-spinner="el-icon-loading"
-			:value="workflowActive"
-			@change="activeChanged"
-			:title="workflowActive ? $locale.baseText('workflowActivator.deactivateWorkflow') : $locale.baseText('workflowActivator.activateWorkflow')"
-			:disabled="disabled || loading"
-			:active-color="getActiveColor"
-			inactive-color="#8899AA">
-		</el-switch>
+		<n8n-tooltip :disabled="!disabled" placement="bottom">
+			<div slot="content">{{ $locale.baseText('workflowActivator.thisWorkflowHasNoTriggerNodes') }}</div>
+			<el-switch
+				v-loading="loading"
+				element-loading-spinner="el-icon-loading"
```

Comment:
Just to be consistent with the rest of code, this line could be moved just before the last line of the bottom.

Reply:
thanks for pointing out. ya I like this order too 😊